libqrexec/exec.c

```diff
@@ -204,20 +204,10 @@ static int qubes_connect(int s, const char *connect_path, const size_t total_pat
     return result;
 }
 
-/* A parsed, mostly-validated RPC command. */
-struct qrexec_parsed_command {
-    /* NULL if and only if we are the fork server.  Otherwise, a NUL-terminated string. */
-    const char *const username;
-    /* Command line.  Never NULL.  NUL-terminated. Does not include "QUBESRPC ". */
-    const char *const command;
-    /* Service descriptor.  Identical to `command` unless the "nogui:" prefix is present, in which case it points
-     * after the colon.  Always points to the start of the service name. */
-    const char *service_descriptor;
-    /* Size of service_descriptor (the service name + argument).  Guaranteed to be <= MAX_SERVICE_NAME_LEN. */
-    size_t const service_descriptor_length;
-};
-
-static int execute_parsed_qubes_rpc_command(const struct qrexec_parsed_command *const command, int *const pid, int *const stdin_fd, int *const stdout_fd, int *const stderr_fd, struct buffer *stdin_buffer);
+static int execute_parsed_qubes_rpc_command(
+    struct qrexec_parsed_command *cmd,
```

Comment:
Is dropping const intentional?

Reply:
No, just some refactoring leftovers. Fixed.